libs/wingsdk/src/cloud/service.ts

```diff
@@ -95,16 +95,25 @@ export class Service extends Resource implements IInflightHost {
     const lines = new Array<string>();
 
     lines.push('"use strict";');
-    lines.push("let $obj;");
-
-    lines.push("async function $initOnce() {");
-    lines.push(`  $obj = $obj || (await (${inflightClient}));`);
-    lines.push("  return $obj;");
+    lines.push("let $stop;");
```

Comment:
style nit: imo it's much easier to read and write multiline templates. It's probably slightly faster at runtime too.

```js
`\
"use strict";
let $stop;
exports.start = async function() {
  if ($stop) {
    throw Error('service already started');
  }
  $stop = (await ((await ${inflightClient})).handle())) ?? (() => {});
};
exports.stop = async function() {
  if (!$stop) {
    throw Error('service not started');
  }
  await $stop();
  $stop = undefined;
};",`
```